<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RERestDishType</class>
 <widget class="QDialog" name="RERestDishType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dish type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="EQLineEdit" name="lePartCode">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="Field" stdset="0">
        <string>f_part</string>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Part</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name, en</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name, am</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Text color</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Queue</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Background color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name, ru</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="EQLineEdit" name="lePartName">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="EQLineEdit" name="leNameAm">
       <property name="Field" stdset="0">
        <string>f_am</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="btnBgColor">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resort/res.qrc">
         <normaloff>:/images/selection.png</normaloff>:/images/selection.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="btnTextColor">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resort/res.qrc">
         <normaloff>:/images/selection.png</normaloff>:/images/selection.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="EQLineEdit" name="leCode">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="PrimaryKey" stdset="0">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_id</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="EQLineEdit" name="leNameEn">
       <property name="Field" stdset="0">
        <string>f_en</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="EColorLineEdit" name="leTextColor">
       <property name="Field" stdset="0">
        <string>f_textColor</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="EColorLineEdit" name="leBgColor">
       <property name="Field" stdset="0">
        <string>f_bgColor</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="EQLineEdit" name="leNameRu">
       <property name="Field" stdset="0">
        <string>f_ru</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>59</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>59</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="EQLineEdit" name="leQueue">
       <property name="Field" stdset="0">
        <string>f_queue</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="EQCheckBox" name="chActive">
       <property name="text">
        <string/>
       </property>
       <property name="Field" stdset="0">
        <string>f_active</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">eqcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>EColorLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ecolorlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leCode</tabstop>
  <tabstop>lePartCode</tabstop>
  <tabstop>lePartName</tabstop>
  <tabstop>leNameAm</tabstop>
  <tabstop>leNameEn</tabstop>
  <tabstop>leNameRu</tabstop>
  <tabstop>leBgColor</tabstop>
  <tabstop>leTextColor</tabstop>
  <tabstop>leQueue</tabstop>
  <tabstop>chActive</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnTextColor</tabstop>
  <tabstop>btnBgColor</tabstop>
 </tabstops>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
